server/src/controllers/user.controllers/create.user.ts

```diff
@@ -21,6 +20,10 @@ const createUser = async (user: IUserModel, next: NextFunction): Promise<IUserMo
             posts,
         }: IUserModel = user;
 
+        if (!email || !fullName || !username || !password) {
+            throw new Error('Some field is empty');
+        }
+
         const emailExist = await User.countDocuments({email});
```

Comment:
try to move all request to backend in separate method in directory dbQuery for example

you can create file userQueries.tsx in order to connect to User Scheme. In this file can be methods emailExist, userExist, findOneAndUpdate etc.

Then any team member could reuse it and don't repeat the same code.